### ISSUE TITLE
Django 1.8 admin AttributeError exception

### ISSUE THREAD

**Issue by kpachnis** [iambrandontaylor/django-admin-sortable#104]
Hi,

In Django 1.8 admin, if the `queryset` method is not implemented, a `AttributeError` exception is raised with the following message:

`'MyModelAdmin' object has no attribute 'queryset'`

A workaround is to always implement the queryset method. For example:

```python
@admin.register(MyModel)
class MyModelAdmin(SortableAdmin):
    def queryset(self, request):
        return super(MyModelAdmin, self).queryset(request)
```

**Comment by alsoicode** [comment]
pull requests are always welcome

**Comment by kpachnis** [comment]
I just made that pull request you mentioned :).

Is tested only against Django 1.8. 

I hope that this will do.

**Comment by alsoicode** [comment]
Thanks. I'll get it pulled down and run tests against it.

**Comment by alsoicode** [comment]
Just released 1.8.3 to address Django 1.8 issues.

**Comment by kpachnis** [comment]
Thanks
